lib/models/stopwatch.dart

```diff
@@ -0,0 +1,19 @@
+/// Stopwatch wrapper that allows stopwatch to be restarted.
+/// Check https://github.com/dwyl/flutter-stopwatch-tutorial#persisting-between-sessions-and-extending-stopwatch-capabilities.
+class TimerEx {
+  final Stopwatch _stopWatch = Stopwatch();
```

Comment:
```suggestion
extension TimerEx on Stopwatch {
  final Stopwatch _stopWatch = Stopwatch();
```

shouldn't this be an extension class?

Reply:
I did try to use the `extension` keyword but I couldn't get it to work the way I needed it to work.
I need the `initialOffset` to get the proper `elapsedMilliseconds` from it so the timer stopwatch can be restarted normally :/

Reply:
Oh I see, because you can only define new behavior/methods. Then I suggest extending `Stopwatch` class

Reply:
Alrighty! I'll change it to `class TimerStopwatch extends Stopwatch` and override the needed methods.

Thanks!